sw/deployer/src/sc64/ff.rs

```diff
@@ -244,6 +361,260 @@ impl FatFs {
     }
 }
 
+
+// TODO, see https://github.com/wfraser/fuse-mt/issues/29
+#[cfg(unix)]
+unsafe impl Send for FatFs {}
+#[cfg(unix)]
+unsafe impl Sync for FatFs {}
```

Comment:
I don't like that. You're using a multi-threaded fuse library just to sidestep the problem by setting 1 max thread and hack an object to be compliant with the fuse_mt API. This is a really bad practice and it will lead to problems down the line.
While FatFs _could_ be used in multi-threaded setting, the connection between PC and SC64 will always be limited by single thread no matter what.
Was the use of `fuse_mt` dictated by the fact, that it exposes "simple" API with path names instead of inodes?

Reply:
Yes, the main motivation for using `fuse_mt` was the simple path based API. 
Even if the connection between PC and SC64 is always single thread, we probably still want to be able to list a dir while reading/writing a big file. This should be already possible with the current implementation because read/write happen in a seperate thread.

I do agree that the current hack to get `FatFs` working with the fuse_mt API is not ideal. I don't know enough about rust to resolve it properly though. I am open for any suggestions